front-end/src/Header.css

```diff
@@ -0,0 +1,6 @@
+h1 {
+    margin-top: 10;
+    padding: 0;
```

Comment:
One more thing: when specifying margins, padding, or really anything involving numbers and positioning, we need to specify what this `10` is--is it pixels? Is it a percentage of the parent element? Or is it relative to the font-size? Without some sort of specification, the styling will be ignored.